packages/ai/src/bootstrap/main.ts

```diff
@@ -8,14 +8,29 @@ export interface AIGatewayContext {
 export interface AIGatewayConfig {
   api: NetlifyAPI
   env: Record<string, { sources: string[]; value: string }>
-  options: { offline?: boolean; offlineEnv?: boolean }
-  site: { id?: string }
+  siteId: string | undefined
   siteUrl: string | undefined
```

Comment:
nit: This would be aligned with other primitives in this monorepo.

```suggestion
  siteURL: string | undefined
```